documentation/quick-start.md

```diff
@@ -0,0 +1,236 @@
+		Authorization: getTokenFromVault(),
+	},
+};
+
+const client = got.extend(options);
+
+export default client;
+```
+
+Some noticeable common options are:
+- [`searchParams`](2-options.md#searchparams): A query string object.
+- [`prefixUrl`](2-options.md#prefixurl): Prepended to query paths. Paths must be relative to prefix, i.e. not begin with a `/`.
+- [`method`](2-options.md#method): The HTTP method name.
+- [`headers`](2-options.md#headers): Query headers.
+- [`json`](2-options.md#json): JSON body.
+- [`form`](2-options.md#form): A form query string object.
+
+See documentation for other [options](2-options.md#options).
+
+## Errors
+
+Both Promise and Stream APIs throw errors with metadata.
+
+```js
+import got from 'got';
+
+try {
+	const data = await got.get('https://httpbin.org/status/404');
+} catch (error) {
+	console.error(error.message);
+}
+```
+
+```js
+import got from 'got';
+
+const stream = got.stream
+	.get('https://httpbin.org/status/404')
+	.once('error', error => {
+		console.error(error.message)
+	});
+```
+
+## Miscellaneous
+
+The HTTP method name can also be given as an option, this may be convenient when it is known only at runtime:
+
+```js
+import got from 'got';
+
+const url = 'https://httpbin.org/anything';
+
+const method = 'POST';
+
+const options = {
+	method,
+	json: {
+		documentName: 'Quick Start',
+	},
+};
+
+const data = await got(url, options);
+```
+
+For most apps, HTTP clients just do `GET` and `POST` queries (`PUT`, `PATCH` or `DELETE` methods work similarly).
+The following sections will give some pointers to more advanced usage.
+
+### Timeouts
+
+By default, requests have no timeout. It is a good practice to set one:
+
+```js
+import got from 'got';
+
+const options = {
+	timeout: {
+		request: 10000,
+	},
+};
+
+const client = got.extend(options);
+
+export default client;
+```
+
+The above sets a global timeout of 10000 milliseconds for all requests issued by the exported `client`. Like all options, timeouts can also be set at the request level. See the [`timeout` option](6-timeout.md#timeout-options).
+
+### Retries
+
+A failed request is retried twice, retry policy may be tuned with a [`retry`](7-retry.md#retry) options object.
+
+```js
+import got from 'got';
+
+const options = {
+	retry: {
+		limit: 5,
+		errorCodes: [
+			'ETIMEDOUT'
+		],
+	},
+};
+```
+
+Retries with stream are a little trickier, see [`stream.on('retry', …)`](3-streams.md#streamonretry-).
+
+### Hooks
+
+Hooks are custom functions called on some request events:
+
+```js
+import got from 'got';
+
+const logRetry = (error, retryCount) => {
+	console.error(`Retrying after error ${error.code}, retry #: ${retryCount}`);
+};
+
+const options = {
+	hooks: {
+		beforeRetry: [
+			logRetry,
+		],
+	},
+};
+
+const client = got.extend(options);
+
+export default client;
+```
+
+*Note that handlers are given as arrays*, thus multiple handlers can be given. See documentation for other possible [hooks](9-hooks.md#hooks-api).
```

Comment:
```suggestion
*Note that hooks are given as arrays*, thus multiple hooks can be given. See documentation for other possible [hooks](9-hooks.md#hooks-api).
```

Not to confuse with actual handlers: https://github.com/sindresorhus/got/blob/main/documentation/10-instances.md#gotextendoptions-instances

Reply:
Yes this is clear.